qutebrowser/misc/utilcmds.py

```diff
@@ -247,3 +247,33 @@ def log_capacity(capacity: int):
         raise cmdexc.CommandError("Can't set a negative log capacity!")
     else:
         log.ram_handler.change_log_capacity(capacity)
+
+
+@cmdutils.register(debug=True, name='debug-loglevel')
+def log_level(level=""):
+    """Change the log level for console logging.
+
+    Args:
+       level: log level for console log.
+    """
+    acceptable_levels=['critical','error','info','debug','warning','vdebug']
+    if level in acceptable_levels:
+        log.change_loglevel(level)
+    else:
+        raise cmdexc.CommandError("Invalid argument, choose from ('critical', 'error', 'warning', 'info', 'debug' or 'vdebug')!")
+
+
+@cmdutils.register(debug=True, name='debug-logfilter')
+def log_filter(filter_names):
+    """Change the log filter for console logging.
+
+    Args:
+       filter_names: Filter names  for the console log.
+    """
+    acceptable_filters=['statusbar','completion','destroy','modes','webview','mouse','misc','url','procs','commands','init','signals','hints','keyboard','downloads','js','qt','rfc6266','ipc','shlexer','save','message','config','sessions']
```

Comment:
We probably need to duplicate the log names *somewhere* (as it isn't a list or dict already in `log.py`), but I'd prefer having a `LOGGER_NAMES = [...]` or so in `log.py` and then using this (also via `@cmdutils.argument` like above).